stream/src/main/scala/org/apache/pekko/stream/impl/fusing/CollectWhile.scala

```diff
@@ -43,18 +44,17 @@ private[pekko] final class CollectWhile[In, Out](pf: PartialFunction[In, Out]) e
       private lazy val decider = inheritedAttributes.mandatoryAttribute[SupervisionStrategy].decider
       import Collect.NotApplied
 
+      @nowarn("msg=Any")
       override final def onPush(): Unit =
         try {
           // 1. `applyOrElse` is faster than (`pf.isDefinedAt` and then `pf.apply`)
           // 2. using reference comparing here instead of pattern matching can generate less and quicker bytecode,
           //   eg: just a simple `IF_ACMPNE`, and you can find the same trick in `Collect` operator.
           //   If you interest, you can check the associated PR for this change and the
           //   current implementation of `scala.collection.IterableOnceOps.collectFirst`.
```

Comment:
Will these comments be outdated after this PR?

Reply:
I think the it's still apply, maybe drop the :
>//   If you interest, you can check the associated PR for this change and the current implementation of `scala.collection.IterableOnceOps.collectFirst`.